admin/scripts/test-release.sh

```diff
@@ -8,7 +8,7 @@
 set -euo pipefail
 
 CUSTOM_REGISTRY_URL="http://localhost:4873"
-NEW_VERSION="$(node -p "require('./packages/docusaurus/package.json').version").NEW"
+NEW_VERSION="$(node -p "require('./packages/docusaurus/package.json').version")-NEW"
```

Comment:
copy of minor fix I had to do in the v2 branch here: https://github.com/facebook/docusaurus/pull/7919